app/scripts/sentry-install.js

```diff
@@ -1,7 +1,10 @@
 import setupSentry from './lib/setupSentry';
 
+// The root compartment will populate this with hooks
+global.sentryHooks = {};
```

Comment:
This file is run before LavaMoat, so it's in the "initial compartment", i.e. the initial execution environment of the realm/JS process.

Reply:
:+1: